src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -480,6 +481,10 @@ public static String getUnknownCommandMessage(final Set<Class<?>> classes, final
         int bestDistance = Integer.MAX_VALUE;
         int bestN = 0;
 
+        final String deprecationMessage = DeprecatedArtifactsRegistry.getToolDeprecationInfo(command);
```

Comment:
Please, make this configurable outside the `getUnknownCommandMessage` to override by downstream toolkits (if someone is trying to run `IndelRealignment` on my toolkit, I would like to throw an error saying that it does not exists but not that it is not included in GATK4 but I would like still to have the code for searching for similar commands...

Reply:
Done.

Reply:
Thanks!